tool/README.md

```diff
@@ -213,3 +213,34 @@ git push -u origin $DEVTOOLS_NEXT_BRANCH
 From the git GUI tool or from github.com directly:
 1. Create a PR.
 2. Receive an LGTM, squash and commit.
+
+## Debug Logs
+
+Debug logs found in `Settings > Copy Logs` can are save such that they can be read by (lnav)[https://lnav.org/]
```

Comment:
r/save/saved
add period at the end of the sentence

Reply:
"Debug logs found in `Settings > Copy Logs` are saved such that they can be read by (lnav)[https://lnav.org/]."